internal/provider/project_resource.go

```diff
@@ -77,6 +91,40 @@ func (r *projectResource) Schema(_ context.Context, _ resource.SchemaRequest, re
 				Computed: true,
 				Optional: true,
 			},
+			"feature_naming": schema.SingleNestedAttribute{
+				Description: "Optional feature naming pattern applied to all features created in this project.",
+				Optional:    true,
+				Attributes: map[string]schema.Attribute{
+					"pattern": schema.StringAttribute{
+						Description: "A JavaScript regular expression pattern, without the start and end delimiters.",
+						Required:    true,
+					},
+					"example": schema.StringAttribute{
```

Comment:
Does this do anything useful for the user? From reading this description I would guess I never care about this